nearswap/src/lib.rs

```diff
@@ -227,6 +230,8 @@ impl NearSwap {
         let (mut p, tokens_out) = self._price_n2t_in(&token, ynear);
         assert_min_buy(tokens_out, min_tokens);
         self._swap_n2t(&mut p, ynear, &token, tokens_out);
+        d.update_storage(start_storage);
+        self.set_deposit(&user, &d);
```

Comment:
let's put lines 223,224,233 and 234 on one helper function. The deposit (`d`) is not used in between.

Reply:
ok